sdg/components.py

```diff
@@ -5,6 +5,26 @@
 
 IMAGE = "quay.io/tcoufal/ilab-sdg:latest"
 
+@dsl.component(base_image=IMAGE)
+def preflight_check_op(
+    repo_branch: str,
+    repo_pr: Optional[int],
+):
+    from os import getenv
+
+    if (not repo_branch) and (repo_pr is None or repo_pr <= 0 ):
+        raise Exception("Both taxonomy repo branch and taxonomy pull request number cannot be empty")
```

Comment:
This gets checked when you try to create the run. It will give an error if you try to proceed without a PR or branch. 
`Failed to create a new run: Failed to generate the ExecutionSpec: invalid pipeline job inputs: Invalid input error: input parameter repo_pr requires type double or integer, but the parameter value is not of number value type`

Can remove this check